helmfile/values/grafana-user.yaml.gotmpl

```diff
@@ -64,13 +64,14 @@ grafana.ini:
     enabled: true
     client_id: grafana
     client_secret: {{ .Values.grafana.clientSecret }}
-    scopes: profile email openid
+    scopes: profile email openid groups
```

Comment:
Can this scope be used when using any other OIDC provider than Okta?

Reply:
In the ideal world, yes - but some don't do it as scopes but instead as claims.  It shouldn't break any other provider implementation but just not work as expected. So perhaps having the scopes as a conditional dependent on  provider? ie. if statement

Reply:
Keep it if it still works with the google oidc connector. Make it configurable if you're feeling ambitious :)